### ISSUE TITLE
[pagination][UX/accessibility] make button size uniform

### ISSUE THREAD

**Issue by yanis-git** [docaxess/webcomponents#53]
because button which display 1 or 2 digits does not have the same with
when you click on next button from 1 to 12 for exemple, overall container will change size, and next button will not being aligned with your cursor

https://webcomponents.ipedis.com/pagination/pagination1
